src/backend/templates/tracks.hbs

```diff
@@ -109,47 +110,47 @@
                               </i>
                             </a>
                           </div>
-                          <h4 style="background-color: {{../color}}; margin-top: -15px; padding-top:15px" class="sizeevent event" data-toggle="collapse" data-target="#desc-{{session_id}} .collapse, #desc2-{{session_id}}" aria-expanded="false" aria-controls="desc-{{session_id}}">
-                            {{#if audio}}
+                          <h4 id="event-upper-block" style="background-color: {{../color}};" class="sizeevent event" data-toggle="collapse" data-target="#desc-{{session_id}} .collapse, #desc2-{{session_id}}" aria-expanded="false" aria-controls="desc-{{session_id}}">
+                             {{#if audio}}
                                <i class="fa fa-music" aria-hidden="true" style="margin-left: 10px;"></i>
                             {{/if}}
-			                      <div id="desc2-{{session_id}}" class="collapse in"	style="background-color: {{../color}}; margin-top: 0px;">
-                              <div class="row">
-                                <div class="col-md-6 col-xs-12 speakers-list">
-  				                      {{#speakers_list}}
-                                <p style="margin-right:10px; float:left" class="session-speakers">
-                                  {{#if photo}}
-                                      <img onError="this.onerror=null;this.src='./images/avatar.png';" data-original="{{thumb}}" class="lazy card-img-top" style="width:5rem; height:5rem; border-radius:50%;"/>
-                                    {{else}}
-                                      <img class="card-img-top" src="./images/avatar.png" style="width:5rem; height:5rem; border-radius:50%;"/>
-                                  {{/if}}
-                                </p>
-  		             	            {{/speakers_list}}
-                                </div>
-                                <div class="col-md-6">
-  		             	            <ul style="padding:0;font-size:15px;">
-  		             	              {{#speakers_list}}
-  		             	                {{name}}{{#if organisation}}&nbsp;({{organisation}}){{/if}}{{#if @last}}{{else}},{{/if}}
-  		             	              {{/speakers_list}}
-  		             	            </ul>
-  		             	            <ul style="padding:0;font-size:15px;">{{type}}</ul>
-                                </div>
-		                          </div>
-                            </div>
+			                      <div id="desc2-{{session_id}}" class="collapse in" style="background-color: {{../color}}; margin-top:0px;">
+                                      <div class="row">
+                                          <div class="col-md-6 col-xs-12 speakers-list">
+				                      {{#speakers_list}}
+				                      <p style="margin-right:10px; float:left" class="session-speakers">
```

Comment:
would have been good if you didn't add further whitespace pollution
(the file is already pretty whitespace polluted)